Dockerfile-grocy

```diff
@@ -47,7 +58,7 @@ RUN     apk update && \
 
 # run php composer.phar with -vvv for extra debug information
 RUN cd /var/www/html && \
-        php composer.phar --working-dir=/www/ -n install && \
+        COMPOSER_AUTH="{\"github-oauth\": {\"github.com\": \"${GITHUB_API_TOKEN}\"}}" php composer.phar --working-dir=/www/ -n install && \
```

Comment:
:warning: This doesn't yet function correctly if `GITHUB_API_TOKEN` is not set.  A shell ternary operator would help here.